src/tabcenter.css

```diff
@@ -363,23 +406,33 @@ img, .tab *:not(.clickable) {
     background-position: center center;
     background-repeat: no-repeat;
     background-size: 12px;
+    z-index: 0;
 }
 
 .tab-close {
     display: none;
 }
 
-.tab:not(.pinned):hover > .tab-close {
+.tab:not(.pinned) > .tab-close {
+    position: absolute;
     display: block;
     width: 24px;
     height: 24px;
-    margin-left: 8px;
+    top: 0;
+    bottom: 0;
+    right: 12px;
+    margin: auto;
```

Comment:
How about top: 50% and transform: translateY(-50%).
If my comment is useless feel free to discard it.